lib/logger2.ts

```diff
@@ -0,0 +1,199 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import {Config} from './configParser';
+
+let useChalk = false;
```

Comment:
What about consolidating all the if statements to up here:

var printRed();
var printYellow();
var printGrey(); 
```js
try {
  chalk = require('chalk');
  printRed = chalk.red;
  // etc
} catch (e) {
  printRed = printYellow = printGrey = function(msg) { return msg; };
}
```